README.md

```diff
@@ -2,73 +2,250 @@
 
 The best way to handle your search bar documentation with [MeiliSearch](https://github.com/meilisearch/MeiliSearch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+To integrate a search bar to your documentation and use this library, you need to scrap your website contents and to push it into a MeiliSearch instance.
+
+Hopefully, we provide all the tools needed and [guide you into these different steps](https://docs.meilisearch.com/resources/howtos/search_bar_for_docs.html) 🚀
+
+As a first introduction, you might only want to test this library without connecting it to your website.<br>
+You can do it by running the playground provided in this repository:
+
+```bash
+$ yarn install
+$ yarn serve
+```
+
+The data comes from MeiliSearch documentation.<br>
+Type `create an indxe` to live the MeiliSearch experience.
+
+### Basic Usage
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/docs-searchbar.js@{version}/dist/cdn/docs-searchbar.min.css"
+    />
+  </head>
+
+  <body>
+    <input type="search" id="search-bar-input" />
+    <script src="https://cdn.jsdelivr.net/npm/docs-searchbar.js@{version}/dist/cdn/docs-searchbar.min.js"></script>
+    <script>
+      docsSearchBar({
+        hostUrl: "https://mymeilisearch.com",
+        apiKey: "XXX",
+        indexUid: "docs",
+        inputSelector: "#search-bar-input",
+        debug: true // Set debug to true if you want to inspect the dropdown
+      });
+    </script>
+  </body>
+</html>
+```
+
+The `hostUrl` and the `apiKey` fields are the credentials of your MeiliSearch instance.<br>
+`indexUid` is the index identifier into the scraper have pushed your website contents.<br>
```

Comment:
This sentence is not clear for me

Reply:
```suggestion
`indexUid` is the index identifier into which the scraper has added your website contents.<br>
```

Reply:
@eskombro what do you think?

Reply:
And @matboivin 😇

Reply:
I would even like to add a third suggestion in which the scraper is not mandatory 
```suggestion
`indexUid` is the index identifier in your MeiliSearch instance in which your website content is stored.<br>
```

Reply:
I prefer the 3rd one!

Reply:
Same!

Reply:
Peeeeuuuurrrfaict (= perfect with the french accent)